app/data_providers/slate_providers/collection_slate_provider.py

```diff
@@ -32,7 +32,7 @@ async def rank_corpus_items(
             **kwargs,
     ) -> List[CorpusItemModel]:
 
-        if kwargs.get('enable_thompson_sampling'):
+        if kwargs.get('seed_id'):
```

Comment:
Suggestion: I think this variable is doing too much work, because it's determining whether Thompson sampling is enabled, and it's making Thompson sampling consistent. I would use two variables for this.

If you prefer one variable, I would name it more specifically, like `enable_thompson_sampling_with_seed`.